project.clj

```diff
@@ -7,7 +7,9 @@
                  [com.stuartsierra/component "0.3.1"]
                  [ring/ring-jetty-adapter "1.5.0"]
                  [suspendable "0.1.1"]]
-  :aliases {"test-all" ["with-profile" "default:+1.8" "test"]}
+  :aliases {"test-all" ["with-profile" "default:+1.8" "default:+1.9" "default:+1.10" "test"]}
```

Comment:
This should be: `"default:+1.8:+1.9:+1.10"`. The colon separates profiles and executes them sequentially.

Reply:
Thanks I didn't know that